### PR TITLE
Added podspec for CocoaPods integration

### PR DESCRIPTION
**Pull request by radazzouz**
**TODO:**

- [x] Merge #17 (this PR is based off of it, the diff will be way smaller then)
- [x] Publish the integration instructions in `ios/cocoapods.md`

### CODE REVIEW

File: ios/cocoapods.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+  # Add any other dependencies here, including any 3rd party native libraries that you depend on for
+  # React Native.
+end
+```
+
+13. Run `pod install`
+14. Open the newly created workspace: `YourApp.workspace`
+15. Copy `PSPDFKit.framework` into the Pods folder: `YourApp/ios/Pods`
+16. Drag and drop if from the Finder into the `RCTPSPDFKit` group:
```
Comment (steviki) [inline ios/cocoapods.md]: Typo: `if` -> `it`

File: react-native-pspdfkit.podspec [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+Pod::Spec.new do |s|
+  s.name         = "react-native-pspdfkit"
+  s.version      = package['version']
+  s.summary      = "A React Native module for the PSPDFKit library."
+  s.authors		 = ["PSPDFKit"]
+  s.homepage     = "https://github.com/PSPDFKit/react-native#readme"
+  s.license      = package['license']
+  s.platform     = :ios, "9.0"
+  s.module_name  = 'PSPDFKitReactNativeiOS'
+  s.source       = { :git => "https://github.com/PSPDFKit/react-native", :branch => "rad/podspec" }
```
Comment (steviki) [inline react-native-pspdfkit.podspec]: Can we omit the branch here, once this is merged to `master`?

Comment (radazzouz) [inline react-native-pspdfkit.podspec]: Nice catch! Yes. Certainly!

File: ios/cocoapods.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+![Deployment Target](../screenshots/embedding-pspdfkit-pods.png)
+17. Add it to the `react-native-pspdfkit` framework:
+![Deployment Target](../screenshots/adding-to-react-native-pspdfkit.png)
+18. Embed `YourApp/ios/PSPDFKit/PSPDFKit.framework` (not the copy from `YourApp/ios/Pods/PSPDFKit.framework`) by drag and dropping it into the "Embedded Binaries" section of the "YourApp" target (Select "Create groups"). This will also add it to the "Linked Frameworks and Libraries" section:
+![Embedding PSPDFKit](../screenshots/embedding-pspdfkit.png)
+19. Add a PDF by drag and dropping it into your Xcode project (Select "Create groups" and add to target "YourApp"). This will add the document to the "Copy Bundle Resources" build phase: 
+![Adding PDF](../screenshots/adding-pdf.png)
+20. Replace the default component from index.ios.js`` with a simple touch area to present the bundled PDF:
```
Comment (steviki) [inline ios/cocoapods.md]: index.ios.js`` -> \`index.ios.js\`

**Pull request merged**